metaflow/metaflow_config.py

```diff
@@ -45,6 +45,13 @@ def from_conf(name, default=None):
 DEFAULT_DATASTORE = from_conf('METAFLOW_DEFAULT_DATASTORE', 'local')
 DEFAULT_METADATA = from_conf('METAFLOW_DEFAULT_METADATA', 'local')
 
+##
+# KFP configuration
+###
+KFP_SDK_NAMESPACE = from_conf('KFP_SDK_NAMESPACE')
+KFP_SDK_USERID = from_conf('KFP_SDK_USERID')
+KFP_SDK_API_NAMESPACE = from_conf('KFP_SDK_API_NAMESPACE') if from_conf('KFP_SDK_API_NAMESPACE') is not None else "kubeflow"
```

Comment:
as noted before, from_conf already has a default parameter, that'd simplify this line

Reply:
Got it, resolved.

Reply:
Please note it as resolved, after the code is pushed and updated.  But push and update and resolve all issues rather than many pushes.

Then it's nice to note it as resolved.
Then the one giving the feedback can close conversation - as they verify it.

thank you for doing this!

Reply:
Ok sounds good! Unresolving for now.

Reply:
Resolved now.